contents/tutorials/property-filter.md

```diff
@@ -9,83 +9,124 @@ tags:
-
-# 3. Configure the filter to remove selected GeoIP info
-
-This app sets all specified properties on ingested events to `null`, effectively preventing PostHog from collecting information you do not want it to use.
-
-To configure the app to remove selected properties, simply select the blue gear icon and enter the properties you wish to remove.
-
-![PostHog Property Filter](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/property-filter/property-filter-tutorial-2.png)
-
-> **Note:** The GeoIP transformation adds event and person properties (`$set` and `$set_once`), so you'll want to filter those out as well. [See below](#event-and-person-properties).
-
-## The full list of GeoIP properties
-
-...at time of writing are
-
-* $geoip_city_name
-* $geoip_country_name
-* $geoip_country_code
-* $geoip_continent_name
-* $geoip_continent_code
-* $geoip_postal_code
-* $geoip_latitude
-* $geoip_longitude
-* $geoip_time_zone
-* $geoip_subdivision_1_name
-* $geoip_subdivision_1_code
-* $geoip_subdivision_2_name
-* $geoip_subdivision_2_code
-* $geoip_subdivision_3_code
-* $geoip_subdivision_3_name
-
-You can [check the current list in the source code for the app.](https://github.com/PostHog/posthog-plugin-geoip)
-
-## Event and Person properties
-
-The GeoIP app sets [person properties using `$set` and `$set_once`](https://posthog.com/docs/integrate/user-properties). If you want to drop those properties prefix the geoip property name with either `$set` or `$set_once`.
-
-For example to ensure `$geoip_cityname` is _never_ stored on either events or persons you would configure:
-
-* $geoip_city_name (the event property)
-* $set.$geoip_city_name (the person proprty)
-* $set_once.$initial_geoip_city_name (the initial and never updated person property)
-
-## A working example
-
-Below is the full configuration Witty Works uses to filter out unwanted data before it is written to PostHog's event log:
-
-```$geoip_city_name,$geoip_longitude,$geoip_latitude,$ip,$geoip_postal_code,$current_url,$performance_raw,$referrer,$initial_referrer,$pathname```
-
-<NewsletterForm />
\ No newline at end of file
+When collecting data with PostHog, you may sometimes want to avoid collecting certain properties for privacy reasons. For example, you may want to ensure you never collect specific kind of data such as IP addresses or exact locations.
+
+That's where the [Property Filter app](/apps/property-filter), which was created by the team at [Witty Works](https://www.witty.works/), comes in. It gives you fine-grained control over which properties are kept or discarded from your events, helping you balance data collection needs with privacy requirements. Here's how to use it:
+
+## How to set up the Property Filter app
+
+The property filter app works by setting the selected properties to `null` when events are ingested. Here's how to set it up:
+
+1. Navigate to the [data pipeline](https://us.posthog.com/pipeline) tab in PostHog.
```

Comment:
```suggestion
1. Navigate to the [data pipelines](https://us.posthog.com/pipeline) tab in PostHog.
```

This will change imminently lol:

https://github.com/PostHog/posthog/pull/26618